### DISCUSSION TITLE
part-ii/2016-127/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#253] [Question comments]
Part II 2016 Riemann Surfaces: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2016-127/

**Comment by eha1234** [comment]
(b) No
<details><summary>(c)</summary>

R compact, so A finite. If they're essential singularities, you're done. Otherwise, $f$ extends to a holomorphic map from $R \rightarrow \mathbb{C}_\infty$, which is surjective. 
</details>
